src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/GetSemanticTokens_Razor_DirectiveAttributesParametersAsync.semantic.txt

```diff
@@ -2,7 +2,7 @@
 0 0 1 84 0 //razorTransition
 0 1 12 87 0 //razorDirective
 1 0 1 91 0 //markupTagDelimiter
-0 1 10 82 0 //razorTagHelperElement
```

Comment:
Do you think we could add a test for `razorComponentAttribute` as well?

Reply:
Is there a reason you figure `GetSemanticTokens_Razor_ComponentAttributeAsync` doesn't cover that?

Reply:
Hm I might be missing something but in `GetSemanticTokens_Razor_ComponentAttributeAsync` I see tokenTypes of `razorComponentElement` and `razorTagHelperAttribute` but not `razorComponentAttribute`. I figured it might be helpful to have `razorComponentAttribute` there somewhere as well just to cover all bases?

Reply:
Ah, good eye. I had the number right but the label I gave it was wrong (I should probably remove those or automate their inclusion on generation).